src/main/kotlin/blackjack/controller/BlackjackController.kt

```diff
@@ -0,0 +1,65 @@
+package blackjack.controller
+
+import blackjack.domain.BlackjackGame
+import blackjack.domain.BlackjackGame.Companion.BLACKJACK_VALUE
+import blackjack.domain.Cards
+import blackjack.domain.User
+import blackjack.view.enterUserNames
+import blackjack.view.printCardReceiveWant
+import blackjack.view.printLine
+import blackjack.view.printResults
+import blackjack.view.printUserCards
+import blackjack.view.printUserNames
+
+class BlackjackController {
+    fun start() {
+        val userNames = enterUserNames().trim()
+        val blackjackGame = BlackjackGame.from(userNames)
+        printUserNames(userNames)
+        printUserCardInfos(blackjackGame.userCards())
+
+        handOutCards(blackjackGame)
+
+        printGameResults(blackjackGame)
+    }
+
+    private fun printUserCardInfos(userCards: Map<User, Cards>) {
+        for (userCard in userCards) {
+            val user = userCard.key
+            printUserCards(user.name, user.cards)
+        }
+    }
+
+    private fun handOutCards(blackjackGame: BlackjackGame) {
+        val users = blackjackGame.cardReceivePossibleUsers()
+        users.forEach { cardReceive(it, blackjackGame) }
+    }
+
+    private fun cardReceive(user: User, blackjackGame: BlackjackGame) {
+        when (printCardReceiveWant(user.name)) {
+            "y" -> {
+                user.addCard(blackjackGame.handOutCard())
+                printUserCards(user.name, user.cards)
+            }
+
+            "n" -> {
+                user.deckComplete()
+            }
```

Comment:
한줄로 끝나는 경우에는 다음과 같이 하는것을 권해드립니다!

```suggestion
            "n" -> user.deckComplete()
```

https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements

<img width="751" alt="image" src="https://github.com/next-step/kotlin-blackjack/assets/13347548/eacf9c88-aef6-4d6f-925a-e66b8b70d1b5">